versions/3.0.md

```diff
@@ -761,67 +751,83 @@ schema:
 
 Describes a single request body.
 
-The `Content-Type` of the request body must be specified by the `consumes` attribute, either at the [top-level](#oasConsumes) or [operation level](#operationConsumes).
```

Comment:
Rather than delete this, instead indicate that the set of allowed Content-Type values is derived from the media types in the `representations` object in the Request Body Object.